### DISCUSSION TITLE
刚发现这个脚本的时候很惊喜，但是用了几天发现有些原本的功能被脚本影响了

### DISCUSSION THREAD

**Discussion by KuiPer-LTW** [the1812/Bilibili-Evolved#4501] [功能建议 / Ideas]
一：非常难受的一个问题，脚本把长按右键加速给屏蔽了，就像是双击全屏一样，但是没有新的组件补回来
原本的这个功能很方便，长按加速松开还原，就和手机版一样，小米甚至专门学着给自己的视频播放器也加了这个功能

二：夜间模式：能不能再功能里加个开关，现在只能用自定义顶栏或者快捷键才能实现开关功能，在设置里找不方便

三：禁止跳转动态详情：复制文字时很好用，但想打开动态时很麻烦，可以在下方转发评论点赞那里加个按钮，点击打开动态详情

四：直播信息扩充：能不能就是单纯的给那个侧栏加个滚动条，因为原版好看，脚本把他们都挤到一块了
像我这种只有在黄金时间才会超过10关注在直播的情况，平时一两个的时候看着就很不舒服
又或者更简单的，在功能里加个开关，这样我要用的时候再开就行了，在设置里开一不方便二还要刷新，总之用的不舒服

五：我想把油猴上的脚本放到这里面该如何操作？我试着从油猴上下下来载入，但好像识别不了，因为我下了不少关于B站的脚本，所以想着把他们都放到这里面一起，这样油猴的界面能简洁一点

六：能不能再设置里调整组件的顺序位置，强迫症，我每次发现新的有用的脚步都要重新按顺序下载

**Comment by Geeyun-JY3** [comment]
一、你开了`快捷键扩展`吧，在`前进`那里禁用掉就好。
三、点击时间戳打开对应内容的详情是各网站通用做法。
五、做不到，Bilibili Evolved 不是油猴脚本管理器；这事在脚本管理器里排序放在一起不差不多嘛。

**Reply by KuiPer-LTW** [reply]
很棒，一 的问题解决了， 不过建议可以写在安装简介里，和双击快进一样
三 的问题我还真不知道，感谢提醒，我笨比了

**Reply by Geeyun-JY3** [reply]
要写也是写到`快捷键扩展`的简介里。

**Comment by Tinhone** [comment]
二、如果有定时开关夜间模式功能的需求的话，可以安装 `夜间模式计划时段` 组件。如果只是想快速调整的话，可以安装 `设置面板 - "最近使用" 类别` 插件，会把最近调整过的插件置顶。或许可以做个新组件，能让用户自行在功能面板里添加用来开关组件的按钮。
四、我感觉确实可以增加点行间距，也可以再做个插件实现类似原版的样式。
六、我也支持。

**Comment by the1812** [comment]
四：没太明白, 平时一两个的时候会有什么问题吗, 和滚动条有什么关系?
六：不太行, 界面上没有能够提供排序的交互, 顶多在代码里排好, 写成插件装上, 类似那个 `设置面板 - "最近使用" 类别`.

**Reply by KuiPer-LTW** [reply]
六无所谓了
四：因为脚本把那块侧栏变挤了，看着不舒服，B站原版看着就很不错

**Reply by KuiPer-LTW** [reply]
![1](https://github.com/the1812/Bilibili-Evolved/assets/103661658/8a952ea8-c682-4858-bdb8-65a1c38b72a6)
![2](https://github.com/the1812/Bilibili-Evolved/assets/103661658/895e0e51-9918-4693-9f78-72dbdcfe6ab7)

**Reply by KuiPer-LTW** [reply]
B站原版是只能显示10个，所以我说能不能不改变侧栏的样式，只是单纯加个滚动条

**Reply by KuiPer-LTW** [reply]
而且我刚刚才发现，这个脚本不会显示只预约了直播但没关注的主播，就是那个英雄联盟官号，这又是一个问题

**Reply by the1812** [reply]
那你可以搞个 Stylus 样式加大一下间距:
```css
.be-live-list-content {
  gap: 16px !important;
}
```

**Reply by KuiPer-LTW** [reply]
哈哈，您太高看我了，看不懂呀

**Reply by the1812** [reply]
[Stylus](https://chromewebstore.google.com/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne) 也是个浏览器扩展, 装完新建一个样式, 匹配 `t.bilibili.com`, 把我上面发的代码粘贴进去保存就行了, 里面的数字改大改小就是调整间距.
